src/app/core/global/global.service.ts

```diff
@@ -24,4 +31,45 @@ export class GlobalService {
             this.statusBar.stopProcess(process, "Failed to fetch platform data.");
         });
     }
+
+    checkForPlatformUpdates() {
+
+        if (!this.checkingForPlatformUpdate) {
+
+            this.checkingForPlatformUpdate = true;
+
+            const process = this.statusBar.startProcess("Checking for platform updates.");
+
+            this.platformRepository.checkForPlatformUpdates().withLatestFrom(this.localRepository.getUpdateAvailable())
+                .take(1).subscribe((result) => {
+
+                this.checkingForPlatformUpdate = false;
+                this.statusBar.stopProcess(process, "Checking for platform updates is finished.");
```

Comment:
No need for completion status message for this, you can stop the process with an empty string.